ecommerce/static/js/models/course_seats/professional_seat.js

```diff
@@ -5,12 +5,13 @@ define([
     function(_,
              CourseSeat) {
         'use strict';
-
         return CourseSeat.extend({
             defaults: _.extend({}, CourseSeat.prototype.defaults,
                 {
                     certificate_type: 'professional',
-                    id_verification_required: false,
+                    // [COLARAZ_CUSTOM]
+                    // Set the default value of is verification required radio button.
+                    id_verification_required: true,
```

Comment:
Why do we need to set it to true?

Reply:
It is responsible to set the default value of the radio button `Is Verification Required?`

Reply:
That I get, but why true? why not false? since we do not need the id verification.

Reply:
Actually, we are setting its default value to true because it is responsible to set the certificate mode which is shown on the certificates page. If we set its value to `false` the mode will be `no-id-professional` which is not required.
![image](https://user-images.githubusercontent.com/42294172/89256400-74413100-d63d-11ea-901f-4e81513122be.png)